controllers/app.controller.js

```diff
@@ -19,11 +19,32 @@ exports.getReview = (req, res, next) =>
 		.then((review) => res.status(200).send({ review }))
 		.catch((err) => next(err));
 
-exports.getAllReviews = (req, res, next) =>
-	fetchAllReviews()
-		.then((reviews) => res.status(200).send({ reviews }))
-		.catch((err) => next(err));
-
+exports.getAllReviews = (req, res, next) => {
+	const sortByWhiteList = [
+		"title",
+		"designer",
+		"owner",
+		"review_img_url",
+		"review_id",
+		"category",
+		"created_at",
+		"votes",
+	];
```

Comment:
We'd usually expect this to be in the model, then we can promise.reject if the sort by doesn't contain any of these.